frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -85,7 +85,9 @@ export const TestCard = ({
         removeTestFromQueue={removeTestFromQueue}
       />
       <Card
-        className={"list-style-none margin-bottom-1em test-card-container"}
+        className={
+          "list-style-none margin-bottom-1em test-card-container prime-queue-item"
```

Comment:
is `prime-queue-item` being added for only the purpose of selecting from our cypress tests? If so, can we modify this so that we are using the `data-cy` attribute instead?